issue.js

```diff
@@ -1,4 +1,4 @@
-var priorityLabelRegex = /^p([1-9])$/;
+var priorityLabelRegex = /^p([0-9]+)$/;
```

Comment:
This change also means that `p0`s can be rebalanced now right? Is that something we want?

Reply:
Oooh, good point. No.

Reply:
Fixed!